openstack/neutron/templates/secret-nsxv3-logstash.yaml

```diff
@@ -7,5 +7,5 @@ metadata:
     application: {{ .Release.Name }}
 type: Opaque
 data:
-  NEUTRON_DB_PASSWORD: {{ coalesce .Values.dbPassword .Values.global.dbPassword .Values.mariadb.root_password | required ".Values.mariadb.root_password is required!" | b64enc }}
+  NEUTRON_DB_PASSWORD: {{ coalesce .Values.dbPassword .Values.global.dbPassword .Values.mariadb.root_password | required ".Values.mariadb.root_password is required!" | include "resolve_secret" | b64enc }}
```

Comment:
No need to change, it should automatically be resolved as it is a plain secret value.